### PR TITLE
Updated information about the service wrapper due to the change from …

### PR DESCRIPTION
**Pull request by ljo**
…Tanuki to YAJSW service wrapper.

### CODE REVIEW

File: data/advanced-installation.xml [hunk 1]
```diff
@@ -84,70 +84,54 @@ Setting admin user password...
...
-                                <ulink url="http://wrapper.tanukisoftware.com/doc/english/launch-nix-boot-solaris.html">Installing The Application To Start on Reboot (Solaris)</ulink>
-                            </para>
-                        </listitem>
-                    </itemizedlist>
+		  If your system supports systemd you can run the
+		  service wrapper as a non-privileged user. You will
+		  be notified on choosing systemd non-privileged when
+		  running the service installer:
+		  <synopsis>tools/yajsw/bin/installDaemon.sh</synopsis>
+		  You will also also be notified to removed it if you
```
Comment (joewiz) [inline data/advanced-installation.xml]: typo: removed > remove

File: data/tuning.xml [hunk 2]
```diff
@@ -55,10 +46,9 @@ vmoptions.mac=-Xdock:name="eXist-db" -Xdock:icon="icon.png" -Dapple.laf.useScree
             <para>On <emphasis>Windows</emphasis>, the <option>-Xmx</option> settings is done in the
                 main .bat files, .e.g. <filename>EXIST_HOME\bin\startup.bat</filename>.</para>
             <para>If you launch <emphasis>eXist-db as a service</emphasis>, all Java settings will be
-                controlled by the service wrapper. In this case, the file to edit is <filename>EXIST_HOME/tools/wrapper/conf/wrapper.conf</filename>. Search for the following
-                lines:</para>
+                controlled by the service wrapper. In this case, the file to edit is <filename>EXIST_HOME/tools/yajsw/conf/wrapper.conf</filename>. The installer sets the memory you chose at install time, but if you want to change it afterwards search for the line:</para>
             <synopsis># Maximum Java Heap Size (in MB)
-wrapper.java.maxmemory=512<!--was 128 - should this not be 1024?-->
+wrapper.java.maxmemory=1024<!--was 128 - should this not be 1024?-->
```
Comment (joewiz) [inline data/tuning.xml]: comment can be removed now, no?

Comment (ljo) [inline data/tuning.xml]: Thanks, yes, it can.

File: data/advanced-installation.xml (hunk 1 above)
Comment (ljo) [inline data/advanced-installation.xml]: Thanks, fixed.

**Pull request merged**